OJP/OJP_StopEvents.xsd

```diff
@@ -119,9 +119,9 @@
 					<xs:documentation>Whether operating day information of this stop event should be included in the response.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="IncludeRealtimeData" type="xs:boolean" default="false" minOccurs="0">
+			<xs:element name="UseRealtimeData" type="UseRealtimeDataEnumeration" default="none" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Whether realtime information of this stop event should be included in the response.</xs:documentation>
+					<xs:documentation>Whether realtime information of this stop event should be used in the response. Default is "none"</xs:documentation>
```

Comment:
Why would it be "none" here? Departure display mode sounds like a proper default for me.

Reply:
changed